src/comon/Sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,67 @@
+import { memo } from 'react';
+import {
+  StyledBox,
+  ActiveLink,
+  ActiveText,
+  DefaultLink,
+  DefaultText,
+} from './Sidebar.styled';
+import { ICONS } from '../../ui-kit/icons';
+import { useSidebarContext } from './SidebarContext';
+
+const Sidebar = () => {
+  const curPath = 'main'; // the path will be obtained from routing
+
+  const { showText } = useSidebarContext();
+
+  return (
+    <StyledBox showText={showText}>
+      {curPath === 'main' ? (
+        <ActiveLink showText={showText}>
```

Comment:
Use NavLink from the react-router-dom library, it has built-in active link functionality